apps/fluent-tester/src/TestComponents/StrokeWidth/StrokeWidthTest.tsx

```diff
@@ -6,17 +6,24 @@ import { Stack } from '@fluentui-react-native/stack';
 import { stackStyle } from '../Common/styles';
 import { globalTokens } from '@fluentui-react-native/theme-tokens';
 import { Text } from '@fluentui/react-native';
+import { Theme, useFluentTheme } from '@fluentui-react-native/framework';
+import { getCurrentAppearance } from '@fluentui-react-native/theming-utils';
+import { themedStyleSheet } from '@fluentui-react-native/themed-stylesheet';
 
-const styles = StyleSheet.create({
-  root: {
-    paddingBottom: 16,
-  },
-  exampleStrokeBackground: {
-    height: 100,
-    backgroundColor: '#f0f0f0',
-    display: 'flex',
-    justifyContent: 'center',
-  },
+const getThemedStyles = themedStyleSheet((t: Theme) => {
+  const isLightMode = getCurrentAppearance(t.host.appearance, 'light') === 'light';
+  return {
+    root: {
+      paddingBottom: 16,
+    },
+    exampleStrokeBackground: {
+      height: 100,
+      backgroundColor: isLightMode ? globalTokens.color.grey80 : globalTokens.color.grey10,
```

Comment:
nit: alias token?

Reply:
I couldn't find an alias token that worked like I wanted to for iOS/macOS and dark/light mode...the next thing on my plate is iOS color tokens, so maybe once that's done there will be a color I can use that works for all platforms. Is there overall a preference to refer to alias tokens over global tokens?

Reply:
For color, yes. It should be pretty rare to reference global colors esp if you're using neutrals. At the moment this test won't show well on HC or win32 themes, where alias tokens would take care of that.

Reply:
It's probably not a blocker though since it's only a test app and for showing tokens.

Reply:
Found one that works okay. Will update to corner radius tokens PR as well. Updated the screenshots in the description